docs/docs/spec.md

```diff
@@ -440,15 +447,16 @@ usage, this section covers only how to determine its [nullness operator].
 To determine the nullness operator, apply the following rules in order. Once one
 condition is met, skip the remaining conditions.
 
--   If the type usage is the type of the field corresponding to an enum
-    constant, its nullness operator is `MINUS_NULL`.
+-   If the type usage is in an intrinsically non-null locations listed earlier, its nullness operator is `MINUS_NULL`.
```

Comment:
What isn't nice about this is that the "intrinsically non-nullable locations" given earlier was in non-normative text, so referring to it in normative text seems bad.

Reply:
Oh, good point.

Maybe the most technically correct thing would be to add an entry about enum-constant _types_ to "However, the type-use annotation is unrecognized in any of the following cases," presumably moving the non-normative comment about enum constants themselves up to there. Then we at least would clearly not be saying that our rules apply to enum-constant types.

We could then still include the text you have here, but it too could be a non-normative comment.

Really, though, I think the chance of _practical_ problems from all this is approximately zero :) It's worth getting right if we can, but if you don't see an approach that you think works well, then I'm fine with what you have or anything else reasonable.

Reply:
I tried doing what you suggested. I highlighted that this is only for *recognized* locations, so the previous rules didn't make sense, as they are unrecognized.